Folly/folly/portability/Builtins.h

```diff
@@ -100,7 +100,7 @@ FOLLY_ALWAYS_INLINE int __builtin_ffsl(long x) {
   return __builtin_ffs(int(x));
 }
 
-#if defined(_M_IX86) || defined(_M_ARM)
+#if defined(_M_IX86) || defined(_M_ARM) || defined(_M_ARM64) // Non-amd64 platforms
```

Comment:
>// Non-amd64 platforms [](start = 61, length = 22)

FWIW, if the intention is as the comment says, could this be abbreviated to '#ifndef _M_X64' or '#ifndef _M_AMD64'?

Reply:
Maybe, but that's assuming we know all possible _M_* possibilities.
This one is less disruptive and more likely to be accepted by Facebook (parallel PR already sent).